src/censure.jl

```diff
@@ -0,0 +1,236 @@
+abstract BiFilter
+
+type BoxFilter <: BiFilter
+
+	scale :: Integer
```

Comment:
Only just noticed now, but there are a few indentation issues in this file. Not a big deal, but FYI 4 spaces is the convention.